packages/discord.js/src/structures/Presence.js

```diff
@@ -389,6 +391,17 @@ class RichPresenceAssets {
 
     return this.activity.presence.client.rest.cdn.appAsset(this.activity.applicationId, this.largeImage, options);
   }
+
+  /**
+   * Tests whether an image key is a URL or not
+   * @private
+   * @param {string} imageKey The image key to test
+   * @returns {boolean}
+   */
+  static _isURLImage(imageKey) {
+    const URLRegex = /(https?|ftp|file):\/\//gm;
```

Comment:
maybe export this regex?

Reply:
I don't see the use

Reply:
Other patterns are exported in other classes and the function isn't public so I think the regex should be so users can use this if they need to